### ISSUE TITLE
Delay toast presentation until Toaster is mounted

### ISSUE THREAD

**Issue by justinkwaugh** [wobsoriano/svelte-sonner#120]
### Describe the feature

Currently if I have an error in a +page.ts load function I might want to show a toast, but I cannot call toast() there because the Toaster component may not yet be mounted.  It would nice if calls to toast() prior to Toaster being mounted would be cached and then run onMount within the Toaster component.

**Comment by demetrius-mp** [comment]
any updates on this?

**Comment by FernandoArteaga** [comment]
You can return the text you want to show in the `toast()` in the load's `data` and display the `data.message` using  `onMount()`

**Comment by justinkwaugh** [comment]
Yes, it's not that the issue cannot be worked around, it's just that it would be nicer if it was handled internally to the library.
